### PR TITLE
Change "--unpack-dir" to use glob pattern

### PR DESCRIPTION
**Pull request by hmak3d**
### TL;DR

This patch allows you to exclude multiple directories from asar packing
```bash
$ asar pack app app.asar --unpack-dir "{**/x1,**/x2,z4/w1}"
````
can be used to exclude multiple directories.

### Problem addressed

Previously, `--unpack-dir` only did prefix match against literal string.  Coupled with [commander limitation that options can only occur once](https://github.com/tj/commander.js/blob/v2.9.0/index.js#L411), this means that you cannot exclude more than one directory from asar packing.

### PR changes

This patch makes `--unpack-dir` behave more like `--unpack` and accept glob patterns ... so you can specify more than one directory using brace expansion (see [minimatch](https://www.npmjs.com/package/minimatch)).

For backwards compatibility, `--unpack-dir` will still do literal prefix match \[if there's no glob match\].

### Notes

I've also posted another PR on the official repo.  If they accept it, we won't need to maintain this fork.

### CODE REVIEW

**Comment by hmak3d** [comment]
This PR is now obsolete.  The changes have been folded into the original repo under release [0.9.1](https://github.com/atom/asar/tree/v0.9.1#excluding-multiple-resources-from-being-packed).
